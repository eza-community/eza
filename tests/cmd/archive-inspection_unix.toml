bin.name = "eza"
args = "tests/itest --inspect-archives --recurse"
