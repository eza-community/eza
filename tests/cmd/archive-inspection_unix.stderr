[numeric field was not a number: >...]
